.github/workflows/foreman.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: Foreman (non-blocking)
+
+on:
+  - pull_request
+
+env:
+  RAILS_ENV: test
+  DATABASE_URL: postgresql://postgres:password@localhost/test
+  DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
+  BUNDLE_WITHOUT: "console:development:journald"
+
+jobs:
+  test:
+    name: "Foreman develop with Ruby ${{ matrix.ruby }} and Node ${{ matrix.node }}"
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: 'postgres:12'
+        ports: ['5432:5432']
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        env:
+          POSTGRES_PASSWORD: password
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.7"
+          - "3.0"
+        node:
+          - "12"
+    steps:
+      - run: sudo apt-get update
+      - run: sudo apt-get install build-essential libcurl4-openssl-dev zlib1g-dev libpq-dev libvirt-dev
+      - name: "Check out Foreman develop"
+        uses: actions/checkout@v4
+        with:
+          repository: ofedoren/foreman
+          ref: "develop"
```

Comment:
I'd think this shouldn't be needed

Reply:
Oh, my bad, leftover from local testing.